hw/formal/data/common_fpv_cfg.hjson

```diff
@@ -0,0 +1,42 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  flow:             fpv
+  flow_makefile:    "{proj_root}/hw/formal/data/fpv.mk"
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson",
+                     "{proj_root}/hw/formal/tools/{tool}/{tool}.hjson"]
+
+  // Default directory structure for the output
+  dut:              "{name}"
+  build_dir:        "{scratch_path}/{build_mode}"
+
+  // we rely on Fusesoc to generate the filelist for fpv
+  sv_flist_gen_cmd:  fusesoc
+  fusesoc_core_:     "{eval_cmd} echo \"{fusesoc_core}\" | tr ':' '_'"
+
+  // TODO: switch the tool to formal once the corresponding edalize backend is avaiable
+  sv_flist_gen_opts: ["--cores-root {proj_root}",
+                      "run",
+                      "--tool=icarus",
+                      "--target=formal",
+                      "--build-root={build_dir}",
+                      "--setup {fusesoc_core}"]
+  sv_flist_gen_dir:  "{build_dir}/formal-icarus"
+
+  // Indicate the tool specific helper sources
+  tool_srcs: ["{proj_root}/hw/formal/tools/{tool}/fpv.tcl"]
+
+  // Vars that need to exported to the env
+  exports: [
+    FPV_TOP: {dut}
+    COV:     {cov}
+  ]
+  report_cmd:  "python3 {proj_root}/hw/formal/tools/{tool}/parse-fpv-report.py"
```

Comment:
@msfschaffner you made a suggestion here:
I think line 27-37 should be moved into a jaspergold.cfg similarly as this has been done for synthesis here.
unless the tcl file and parsing script can be shared among all our FPV tools that we are currently using.

My thought here is: we are specifying `  report_cmd:  "python3 {proj_root}/hw/formal/tools/{tool}/parse-fpv-report.py"`, so if other tools uses "dvsim", they should have a `parse-fpv-report.py` under their tool directory. They should also have a run script under their directory called `fpv.tcl`.

Reply:
yeah you are right. for FPV the tool setup will most likely be the same for other tools as well (i.e., one run script and one parser script, and nothing else). let's leave it as it is right now. we can always extend this later if needed.